library/selinux_modules_facts.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+
+# Copyright: (c) 2020, Petr Lautrbach <plautrba@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: selinux_module_facts
```

Comment:
selinux_modules_facts ?

Reply:
> selinux_modules_facts ?

yes - similar to the `service_facts` module - https://docs.ansible.com/ansible/2.9/modules/service_facts_module.html#service-facts-module

Reply:
Oh, I just thought **s** in selinux_module**s**_facts is missing. :p

Reply:
Fixed, thanks!